test/fixtures/circular-example.yaml

```diff
@@ -25,8 +26,11 @@ definitions:
         properties:
           data:
             $ref: '#/definitions/User'
+    default:
+      id: ORCL
```

Comment:
We have to supply a defualt excluding the circular reference so that faker won't randomly expand optional user making the test flaky.